packages/frontend/src/features/common/styles/constans/fontWeight.tsx

```diff
@@ -1,10 +1,8 @@
 export enum fontWeight {
-    light = 300,
-    regular = 400,
-    medium = 500,
+    LIGHT = 300,
     REGULAR = 400,
+    MEDIUM = 500,
     SEMI_BOLD = 600,
     BOLD = 700,
-    bold = 700,
-    black = 900,
+    BOLDER = 900,
```

Comment:
Czemu nie black? (to czesto spotykana nazwa)

Reply:
z nazwą Bolder też sie kiedyś spotkałem